vars/neoDeploy.groovy

```diff
@@ -59,16 +59,31 @@ import static com.sap.piper.Prerequisites.checkScript
      * @mandatory for deployMode=warParams
      */
     'runtimeVersion',
-        /**
```

Comment:
[nit-pick] unrelated

Reply:
I would like to keep this. Can't we go by the Scouts Rule: _Leave the campground cleaner than you found it.5_?

Reply:
True, but still I would like to fix that within this PR as the comment part looks weird only for these 3 parameters.

Reply:
I see now that my first sentence can be interpreted two ways. What I meant is I would like to keep it changed, the same as you @srinikitha09

Reply:
Of course we can keep it, I did not say "remove this" and I also did not say "require changes". Comment was also annotated with "nit-pic".

The general rule should be: keep such kind of formatting away since it makes reviewing harder. Do a dedicated reformatting pull request. Even if that are only a few lines. Beside reviewing it makes also factoring-out commits easier and simplifies `git bisect`.

Reply:
> Of course we can keep it, I did not say "remove this" and I also did not say "require changes". Comment was also annotated with "nit-pic".
> 
you are right! I understood. May be I would have mistaken what Oliver meant. My bad.

So, do I have to do it in another dedicated PR or is it fine for now t have it here?

Reply:
keep it ...